projects/angular-cld/src/lib/cloudinary-image.component.ts

```diff
@@ -184,6 +192,6 @@ export class CloudinaryImage
   }
 
   accessibilityModeHandler() {
-    return this.cloudinary.url(this.publicId, {transformation: [this.options, accessibilityEffect[this.accessibility]]});
+    return this.cloudinary.url(this.publicId, {transformation: [this.options, accessibilityEffect[this.accessibility]], accessibility: true, ...SDKAnalyticsConstants});
```

Comment:
We added the SDKAnalyticsconstants to options since in loadImage() the src is generated with options. 
In placeholder and accessibility- if you take a look at the URL we're only passing this.options into transformation so that we generate the same URL but with the accessibility/placeholder transformation appended. That's the reason we need to send in the SDKanalytics in cases we use URL

Reply:
So, we have the options that has the SDKAnalytics in it, we take the transformations out of it, and then we re-add it?

Reply:
Yes

Reply:
Can't we do it the other way around? extract the transformation and the analytics? 
I want to minimize the times we call the Analytics consts, as in the future, we might change things, like dynamically generate some of the values and not from consts, and then we will need to change in 5 places instead of 1